apps/examples/testcase/ta_tc/task_manager/itc/itc_task_manager_main.c

```diff
@@ -146,9 +186,10 @@ void itc_taskmanager_check_dedicate_permission_p(void)
 	TC_SUCCESS_RESULT();
 }
 
-void itc_taskmanager_user1_main(void)
+void itc_task_manager_user1_main(void)
```

Comment:
Please check this commit.
https://github.com/Samsung/TizenRT/commit/611636a54fd1f263b25e8c6e096c3478150cf6ae

Reply:
you should this again as jeongchan said.